_includes/commands/template-cmds.md

```diff
@@ -0,0 +1,10 @@
+
+| Command           | Description                                                                     |
+| ----------------- | ------------------------------------------------------------------------------- |
+| mceInsertTemplate | Inserts a template the value should be the template HTML to process and insert. |
+
+**Example**
+
+```js
+tinymce.activeEditor.execCommand('mceInsertTemplate');
```

Comment:
This example is incorrect, as it needs to be passed a value as per the description. eg:
```
tinymce.activeEditor.execCommand('mceInsertTemplate', false, '<p class="cdate">This will be replaced with the creation date.</p>')
```

Reply:
Updated to:
```
tinymce.activeEditor.execCommand('mceInsertTemplate', false, '<p>This is my template text.</p>');
```